dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -747,6 +748,35 @@ public User updateLoggedInUserMetadata(@ApiParam(hidden = true)@Parameter(hidden
         return dbuser;
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @RolesAllowed({"admin", "curator"})
+    @Path("/user/{userId}/privileges")
+    @Operation(operationId = "addUserPrivileges", description = "Updates the provided userID to admin or curator status, ADMIN or CURATOR only", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
```

Comment:
The API also supports removing user privileges. I would call this `setUserPrivileges`

Reply:
Done.